mlmodel/docs/conf.py

```diff
@@ -1,26 +1,28 @@
-# -*- coding: utf-8 -*-
+# Configuration file for the Sphinx documentation builder.
```

Comment:
Why are there changes in this file since this one is unrelated to the doc generation for the proto files ?

Reply:
I use a newer version of Sphinx, which requires the newer `conf.py` (or causes an error). Part of this process fixes the project name (so that "Core ML Format Reference" appears at the top). I also made changes to use the same extensions as before, and to set the RTD theme.

Reply:
I didn't quite get the question. This is the `conf.py` file in `coremltools/mlmodel/docs`, not `coremltools/docs`. It is now at the same version level as the one in `coremltools/docs`.